starlite/logging/__init__.py

```diff
@@ -6,6 +6,11 @@
 
 from starlite.logging.standard import QueueListenerHandler
 
+try:
```

Comment:
Is there a better way to do this?  I try to import it and set to `None` if it fails.  In the `configure`, we try to run `picologging.dictConfig` if `picologging` is found in the logging class name.

Reply:
This is fine